docs/keystone-auth/using-auth-data-synchronization.md

```diff
@@ -16,7 +16,7 @@
 
 Authentication synchronization is an experimental feature allowing to create Kubernetes auth related entities during user authentication.
 
-As an example, you can take the synchronization of Keystone projects with Kubernetes namespaces. If the user belongs to the project in Keystone, then, when attempting to authenticate in Kubernetes using [k8s-keystone-auth](./using-keystone-webhook-authenticator-and-authorizer.md) webhook for the first time, a Kubernetes namespace will automatically be created for the user.
+As an example, you can take the synchronization of Keystone projects with Kubernetes namespaces. If the user belongs to the project in Keystone, then, when attempting to authenticate in Kubernetes using [k8s-keystone-auth](./keystone-auth/using-keystone-webhook-authenticator-and-authorizer.md) webhook for the first time, a Kubernetes namespace will automatically be created for the user.
```

Comment:
Should it be `../keystone-auth/using-keystone-webhook-authenticator-and-authorizer.md`?

Reply:
@lingxiankong updated with ./using-keystone-webhook-authenticator-and-authorizer.md as they are in same folder.